### ISSUE TITLE
Events that start on one day and end on another aren't displayed on /events list after start date

### ISSUE THREAD

**Issue by amandavisconti** [scholarslab/scholarslab.github.io#1136]
E.g. Halloween 10/28/2024-10/31/2024 event is not visible on 10/30. I made the slug and start date both 10/13 so it would appear (rather than use acurate start-date and end-date YAML)

**Comment by jeremyboggs** [comment]
We might confirm whether this is still an issue after resolving 0e36737ac6a98999809d77e64a6e5f0c7ddb825f.

**Comment by amandavisconti** [comment]
Agree—it's on [the Events Pt 2 issue to see if we can close](https://github.com/scholarslab/scholarslab.github.io/issues/1150)

**Comment by jeremyboggs** [comment]
confirmed that this is fixed.
